scene/gui/rich_text_label.cpp

```diff
@@ -5412,6 +5412,9 @@ void RichTextLabel::append_text(const String &p_bbcode) {
 			push_language(lang);
 			pos = brk_end + 1;
 			tag_stack.push_front("lang");
+		} else if (tag == "br") {
+			add_text("\r");
```

Comment:
Out of curiosity, why is this `\r` and not `\n`?

Reply:
See my previous comment, `\n` start new paragraph (hardcoded as paragraph separator), `\r` just add a break to existing one.